### PR TITLE
additional details to the readme

### PR DESCRIPTION
**Pull request by codeboten**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
...
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1P2xejC7lEkOV_Z-8E0oZPXLK5HOnUPNuRqKP0ZQ5fpg).
+
+## Project Leadership
+
+* [Carter Socha](https://github.com/cartersocha), Lightstep
+* [Juraci Paixão Kröhling](https://github.com/jpkrohling), Grafana Labs
+
+## Contact
+
+* Slack: [#otel-security](https://cloud-native.slack.com/archives/C05A85QC281)
```
Comment (tylerbenson) [inline README.md:19]: Carter just renamed the channel.

```suggestion
* Slack: [#otel-sig-security](https://cloud-native.slack.com/archives/C05A85QC281)
```

**Review by tylerbenson — APPROVED** [review]

**Comment by cartersocha** [comment]
@reyang needs to be added as a maintainer too

**Comment by codeboten** [comment]
> LGTM, once the Slack channel renaming is taken into account.

I have no idea when that rename happened, i created the readme and #otel-security was still there

**Comment by jpkrohling** [comment]
It happened after your opened the PR and before I reviewed it :-)

**Comment by cartersocha** [comment]
@codeboten I did it today. Wanted to wait until we had a confirmed repo name / brand

**Pull request merged**